src/encoded/static/components/app.js

```diff
@@ -1177,20 +1178,16 @@ export default class App extends React.PureComponent {
                     <meta name="twitter:dnt" content="on" />
                     <HTMLTitle {...{ context, currentAction, canonical, status }} />
                     {base ? <base href={base}/> : null}
-                    <link rel="canonical" href={canonical} />
-                    <script data-prop-name="user_details" type="application/json" dangerouslySetInnerHTML={{
-                        __html: jsonScriptEscape(JSON.stringify(JWT.getUserDetails())) /* Kept up-to-date in browser.js */
-                    }}/>
-                    <script data-prop-name="lastCSSBuildTime" type="application/json" dangerouslySetInnerHTML={{ __html: lastCSSBuildTime }}/>
-                    <link rel="stylesheet" href={'/static/css/style.css?build=' + (lastCSSBuildTime || 0)} />
+                    <link rel="stylesheet" href={'/static/css/style.css?build=' + (lastCSSBuildTime || 0)} type="text/css" />
                     <link rel="preconnect" href="https://unpkg.com" />
                     <link rel="preconnect" href="https://fonts.googleapis.com/" />
                     <link rel="preconnect" href="//www.google-analytics.com" />
-                    <link rel="stylesheet" href="https://unpkg.com/rc-tabs@9.6.0/dist/rc-tabs.min.css" />
+                    <link rel="stylesheet" href="https://unpkg.com/rc-tabs@9.6.0/dist/rc-tabs.min.css" type="text/css" />
                     <SEO.CurrentContext {...{ context, hrefParts, baseDomain }} />
-                    <link href="https://fonts.googleapis.com/css?family=Mada:200,300,400,500,600,700,900|Yrsa|Source+Code+Pro:300,400,500,600" rel="stylesheet"/>
-                    <script async type="application/javascript" src={"/static/build/bundle.js?build=" + (lastCSSBuildTime || 0)} charSet="utf-8" />
+                    <link href="https://fonts.googleapis.com/css?family=Mada:200,300,400,500,600,700,900|Yrsa|Source+Code+Pro:300,400,500,600" rel="stylesheet" type="text/css"/>
+                    <script defer type="application/javascript" src={"/static/build/bundle.js?build=" + (lastCSSBuildTime || 0)} charSet="utf-8" />
```

Comment:
Appears to fix flash of unstyled content (FOUC) glitch on current versions of FireFox.

Reply:
Noticing that we're using rel="preconnect" here. That was unsupported by Safari and IE last time I checked, IIRC. I think I remember hearing something about trying to push users towards Chrome & Firefox for cgap; assuming that's the case here, too?

Reply:
Yep for most part. Added the preconnect in recently mostly based on [this article](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/loading-third-party-javascript#use_resource_hints_to_reduce_connection_setup_time); haven't tested w/ IE or Safari so not really sure what happens when unsupported but hoping it degrades gracefully or is ignored by them.